vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -143,12 +143,16 @@ EOF
 }
 
 enableMarinerKata() {
-    # Enable the mshv boot path
-    sudo sed -i -e 's@menuentry "CBL-Mariner"@menuentry "Dom0" {\n    search --no-floppy --set=root --file /EFI/Microsoft/Boot/bootmgfw.efi\n        chainloader /EFI/Microsoft/Boot/bootmgfw.efi\n}\n\nmenuentry "CBL-Mariner"@'  /boot/grub2/grub.cfg
+    echo "Contents of blkid output"
+    export my_blkid=$(blkid)
+    echo $my_blkid
 
-    # kata-osbuilder-generate is responsible for triggering the kata-osbuilder.sh script, which uses
-    # dracut to generate an initrd for the nested VM using binaries from the Mariner host OS.
-    systemctlEnableAndStart kata-osbuilder-generate
+    export boot_uuid=$(sudo grep -o -m 1 '[0-9a-f]\{8\}-[0-9a-f]\{4\}-[0-9a-f]\{4\}-[0-9a-f]\{4\}-[0-9a-f]\{12\}' /boot/efi/boot/grub2/grub.cfg)
```

Comment:
This can be removed **only after** we merge grub2-mkconfig enablement in our core image. There is an additional dependency on a kernel-mshv change which is described here: https://github.com/microsoft/CBL-Mariner/pull/6299#issuecomment-1738246877

Reply:
It is possible we don't merge automatic grub2-mkconfig enablement at all. In that case, there will be a line here calling 

'grub2-mkconfig > /boot/grub2/grub.cfg' 

to put this image on the mkconfig track.

Reply:
Future work item when relevant Mariner work is release: https://microsoft.visualstudio.com/OS/_workitems/edit/46713189